packages/currency/src/chainlink-path-aggregators.ts

```diff
@@ -272,3 +273,71 @@ export function getPath(
   // Get the path
   return route.path(currencyFrom.hash.toLowerCase(), currencyTo.hash.toLowerCase());
 }
+
+/**
+ * These currencies are supported by Chainlink for conversion.
+ */
+export const supportedCurrencies: Record<string, Record<RequestLogicTypes.CURRENCY, string[]>> = {
```

Comment:
```suggestion
export const currenciesWithConversionOracles: Record<string, Record<RequestLogicTypes.CURRENCY, string[]>> = {
```

Reply:
I see what you mean. In the meantime I managed to generate the supportedCurrencies map out of the aggregators map in currency package, using the logic from toolbox.
So I will leave the mapping in advanced-logic but generated it out of the aggregator list in /currency package.
The whole idea behind this small refactor was to stop keeping duplicates and have one source of truth.